### DISCUSSION TITLE
How to know dependent libraries as well as kernel modules for pycuda ?

### DISCUSSION THREAD

**Discussion by rifaterdemsahin** [inducer/pycuda#331] [Q&A]
Docker nvidia/cuda:9.0-devel-ubuntu16.04 ( only way i was able to install pycuda by pip)
python 3.7
pycuda version 2018.1.1
Cuda version 9 ( Azure Container Instance Limitation )

I am getting this error / what should i install not to get the error ?
```
  bpl-subset/bpl_subset/boost/system/error_code.hpp:216:36: warning: ‘pycudaboost::system::native_ecat’ defined but not used [-Wunused-variable]
       static const error_category &  native_ecat    = system_category();
                                      ^
  error: command 'x86_64-linux-gnu-gcc' failed with exit status 1
  ----------------------------------------
  ERROR: Failed building wheel for pycuda

```

**Comment by inducer** [comment]
I think what you pasted there is a warning unrelated to the error that caused your build to fail.
